apps/site/src/app/(home)/sections/Sponsors/Sponsors.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @next/next/no-img-element */
+import Image from "next/image";
+
+import { getSponsors } from "./getSponsors";
+import { client } from "@/lib/sanity/client";
+import imageUrlBuilder from "@sanity/image-url";
+import fishingBoat from "@/assets/images/fishing-boat.png";
+
+import styles from "./Sponsors.module.scss";
+
+const builder = imageUrlBuilder(client);
+
+export default async function Sponsors() {
+	const sponsors = await getSponsors();
+
+	return (
+		<section className="container py-24 md:my-16 relative items-center flex flex-col md:p-8 w-3/4 mx-auto text-center">
+			<h2
+				className={`${styles.title} my-12 font-display sm:text-[3rem] text-[#fffce2] text-4xl text-center`}
+			>
+				Sponsors
+			</h2>
+			<p className="max-w-md mb-12">
+				Interested in sponsoring IrvineHacks 2024? Check out our
+				information above to learn more about our event! For more
+				information, please email us at{" "}
+				<a href="mailto:hack@uci.edu" className="hover:underline">
+					hack@uci.edu
+				</a>
```

Comment:
Nitpick: The link still isn't really distinguishable from the rest of the text unless you hover over it? Maybe keep the underline permanent and then bold on hover?

Reply:
I kept the underline on hover and made the link permanently bold.